api/src/main/java/org/pmiops/workbench/billing/FreeTierBillingService.java

```diff
@@ -195,8 +208,16 @@ private void maybeAlertOnCostThresholds(
       if (currentFraction > threshold) {
         // only alert if we have not done so previously
         if (previousFraction <= threshold) {
-          notificationService.alertUserFreeTierDollarThreshold(
-              user, threshold, currentCost, remainingBalance);
```

Comment:
I missed that we needed expiration date for this alert when I was doing the logic story